integration_tests/features/Propagation.feature

```diff
@@ -50,3 +50,35 @@ Feature: Block Propagation
     Then I receive an error containing 'Orphan block'
     And all nodes are at height 1
 
+  Scenario: Nodes should never sync but keep insync
+    Given I have 2 seed nodes
+    And I have 4 base nodes connected to all seed nodes
```

Comment:
The seed nodes and these 4 base nodes don't seem to take part in this test. Can they be removed/reduced?

Reply:
They can, it was just to provide a small network.